app/assets/javascripts/reducers/active_courses.js

```diff
@@ -0,0 +1,42 @@
+import {
+  RECEIVE_ACTIVE_COURSES,
+  SORT_ACTIVE_COURSES,
+  RECEIVE_CAMPAIGN_ACTIVE_COURSES
+} from '../constants/active_courses';
+import { sortByKey } from '../utils/model_utils';
+
+const initialState = {
+  sort: {
+    key: null,
+    sortKey: null,
+  },
+  courses: [],
+  isLoaded: false,
+};
+
+export default function active_courses(state = initialState, action) {
+  switch (action.type) {
+    case RECEIVE_ACTIVE_COURSES:
+    case RECEIVE_CAMPAIGN_ACTIVE_COURSES: {
+      return {
+        ...state,
+        courses: action.data.courses,
+        isLoaded: true,
+      };
+    }
+    case SORT_ACTIVE_COURSES: {
+      const desc = action.key === state.sort.sortKey;
```

Comment:
As we discussed, this should follow the example of the `articles` reducer for handling default sort order for columns where `desc` should be the default.